pkg/scheduler/api/validation/validation_test.go

```diff
@@ -28,44 +28,55 @@ func TestValidatePolicy(t *testing.T) {
 	tests := []struct {
 		policy   api.Policy
 		expected error
+		name     string
 	}{
 		{
+			name:     "no weight defined in policy",
 			policy:   api.Policy{Priorities: []api.PriorityPolicy{{Name: "NoWeightPriority"}}},
 			expected: errors.New("Priority NoWeightPriority should have a positive weight applied to it or it has overflown"),
 		},
 		{
+			name:     "policy weight is not positive",
 			policy:   api.Policy{Priorities: []api.PriorityPolicy{{Name: "NoWeightPriority", Weight: 0}}},
 			expected: errors.New("Priority NoWeightPriority should have a positive weight applied to it or it has overflown"),
 		},
 		{
+			name:     "valid weight priority",
 			policy:   api.Policy{Priorities: []api.PriorityPolicy{{Name: "WeightPriority", Weight: 2}}},
 			expected: nil,
 		},
 		{
+			name:     "invalide negative weight policy",
 			policy:   api.Policy{Priorities: []api.PriorityPolicy{{Name: "WeightPriority", Weight: -2}}},
 			expected: errors.New("Priority WeightPriority should have a positive weight applied to it or it has overflown"),
 		},
 		{
+			name:     "policy weight exceeds maximum",
 			policy:   api.Policy{Priorities: []api.PriorityPolicy{{Name: "WeightPriority", Weight: api.MaxWeight}}},
 			expected: errors.New("Priority WeightPriority should have a positive weight applied to it or it has overflown"),
 		},
 		{
+			name:     "valid weight in policy extender config",
 			policy:   api.Policy{ExtenderConfigs: []api.ExtenderConfig{{URLPrefix: "http://127.0.0.1:8081/extender", PrioritizeVerb: "prioritize", Weight: 2}}},
 			expected: nil,
 		},
 		{
+			name:     "invalid negative weight in policy extender config",
 			policy:   api.Policy{ExtenderConfigs: []api.ExtenderConfig{{URLPrefix: "http://127.0.0.1:8081/extender", PrioritizeVerb: "prioritize", Weight: -2}}},
 			expected: errors.New("Priority for extender http://127.0.0.1:8081/extender should have a positive weight applied to it"),
 		},
 		{
+			name:     "valid filterverb and url prefix",
 			policy:   api.Policy{ExtenderConfigs: []api.ExtenderConfig{{URLPrefix: "http://127.0.0.1:8081/extender", FilterVerb: "filter"}}},
 			expected: nil,
 		},
 		{
+			name:     "valid preemtverb and urlprefix",
```

Comment:
typo: preemtverb

Reply:
This typo didn't get fixed, but otherwise LGTM.

Reply:
Oh, I saw the comment and thought it was after fixed. I'm blind ...